build.gradle.kts

```diff
@@ -1,3 +1,38 @@
+import io.gitlab.arturbosch.detekt.Detekt
+
 plugins {
-  id("org.jetbrains.kotlin.jvm") version "1.9.0" apply false
+  id("org.jetbrains.kotlin.jvm") version "1.9.0"
```

Comment:
was `apply false` removed because we're using this plugin in this gradle file now?

Reply:
Correct. It's used for the 
```
  kotlin {
    explicitApi()
  }
```
section. It's applied to all subprojects.